libbeat/processors/script/javascript/session.go

```diff
@@ -288,14 +289,28 @@ func newSessionPool(p *goja.Program, c Config) (*sessionPool, error) {
 			s, _ := newSession(p, c, false)
 			return s
 		},
-		C: make(chan *session, c.MaxCachedSessions),
+		C:                  make(chan *session, c.MaxCachedSessions),
+		NewSessionsAllowed: !c.OnlyCachedSessions,
```

Comment:
style nit

For the humans it is a little easier to keep track of things if the config file option, and the struct field and the meaning are the same.  The fact that you have to do a ! here is a code smell.  Any chance this could be refactored so the name is the same throughout and the `false` is the default value?

Reply:
Updated the PR per this feedback!